src/app/index.ts

```diff
@@ -301,19 +301,28 @@ module.exports = yo.extend({
         if (this.project.framework === 'manifest-only') {
           this.fs.copyTpl(this.templatePath(`manifest-only/**`), this.destinationPath(), templateFills);
         }
-        else if (this.project.framework != excelCustomFunctions) {
-          /** Copy the base template */
-          this.fs.copy(this.templatePath(`${language}/base/**`), this.destinationPath(), { globOptions: { ignore: `**/*.placeholder` }});
+        else 
+        {
+          if (this.project.framework === excelCustomFunctions)
+          {
+            /* Copy custom function project template and only the essential files needed from the base directory  */

```

Comment:
Suggestion: in a separate PR, this code could be improved to use helper functions for tasks to copy base template, project template, manifest, files starting with dot, etc.

Reply:
It's hard to tell from the diff but how is custom functions template now different from others? Before there was just a simple else to do the copy. Perhaps there's a simpler refactor to allow it to copy the js\excel-custom-functions-preview template regardless of language (and later we'll add ts template and use language).

Reply:
@akrantz I will incorporate this suggestion into the pending changes in the simply-yo-office-prompting branch